README.md

```diff
@@ -0,0 +1,175 @@
+# UWFE Firmware
+
+This is the firmware monorepo for the UW Formula Electric team. This repository contains vehicle board firmware, HIL board firmware, and our firmware validation testbed.
+
+# Onboarding Instructions
+
+- [OpenProject Access](#open-project)
+- [SSH Key](#ssh-key-set-up)
+- [Vagrant (Windows Only)](#Vagrant-Environment-Set-Up-for-Windows-Users)
+- [Mac OS](#mac-os-set-up)
+- [Linux](#linux-set-up)
+- [Other Resources](#other-resources)
+
+# Open Project
+
+We use Open Project for task management. (todo: add instructions on how to get access lol)
```

Comment:
1. http://208.68.36.87/
2. Create an account with their school email
3. Drop your school email in the new-members thread under #Firmware on Slack

Then Aryan or Jacky will add them to the OpenProject

Reply:
step 1: I think the owenbrake.com/pm is a better link lol

Reply:
wait I never knew that lol